src/tools/rustfmt/tests/source/frontmatter_spaced.rs

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env cargo
+
+
+---   identifier
+[dependencies]
+regex = "1"
+
+---
```

Comment:
Note: I had to strip the trailing whitespace to get around `error[internal]: left behind trailing whitespace` since we aren't formatting these yet.